### DISCUSSION TITLE
Custom date format left side and "until"

### DISCUSSION THREAD

**Discussion by atomicsp** [totaldebug/atomic-calendar-revive#1501] [Feature Requests]
I want to use a date format that looks like this "ddd DD. MMM" for the left side and "DD.MMM" for the "until" part.
Its currently showing it as "dd MMMM DD" and "MMMM DD"

It seems I can't change the date format used under event mode.
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/28091120/9c19dd46-7ea9-41dd-abcf-7fd05deedc9a)

However it look different in the examples images, here it looks to be "DD MMMM":
![image](https://raw.githubusercontent.com/totaldebug/atomic-calendar-revive/master/.github/img/event-mode-no-date.png)

**Comment by marksie1988** [comment]
this will be possible to change in the latest version of the card that i will release soon.
